unittests/tools/test_sarif_parser.py

```diff
@@ -559,3 +559,10 @@ def test_get_fingerprints_hashes(self):
             {"stableResultHash": {"version": 2, "value": "234567900abcd"}},
             get_fingerprints_hashes(data2["fingerprints"]),
         )
+
+    def test_tags_from_result_properties(self):
+        testfile = open(path.join(path.dirname(__file__), "../scans/sarif/taint-python-report.sarif"))
```

Comment:
Will you please ensure pep8 compliance on line length?

Reply:
@rsaiprashanth We're working towards pep8 compliance at 120 character line length - we're not enforcing it for PRs yet but it would be appreciated if you could make this minor change.